vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/daily_cap_frequency_cap.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BAT_ADS_INTERNAL_DAILY_CAMPAIGN_FREQUENCY_CAP_H_
+#define BAT_ADS_INTERNAL_DAILY_CAMPAIGN_FREQUENCY_CAP_H_
+
+#include <string>
+
+#include "bat/ads/internal/frequency_capping/exclusion_rule.h"
+
+namespace ads {
+struct AdInfo;
+class FrequencyCapping;
+
+class DailyCapFrequencyCap : public ExclusionRule {
+ public:
+  explicit DailyCapFrequencyCap(
+      const FrequencyCapping* frequency_capping)
+      : frequency_capping_(frequency_capping) {
+    }
+  virtual ~DailyCapFrequencyCap() = default;
+
+  bool ShouldExclude(
+      const AdInfo& ad) override;
+
+  const std::string& GetLastMessage() const override;
+
+ private:
+  const FrequencyCapping* frequency_capping_;  // NOT OWNED
```

Comment:
Please remove `const`

Reply:
After further discussion, we are changing to `const FrequencyCapping* const frequency_capping_;`